frontend/src/metabase/query_builder/components/filters/modals/InlineCategoryPicker/SimpleCategoryFilterPicker.tsx

```diff
@@ -29,9 +30,15 @@ export function SimpleCategoryFilterPicker({
     onChange(filter.setArguments(newArgs));
   };
 
+  const hasShortOptions = !options.find(
+    option => String(option).length > LONG_OPTION_LENGTH,
```

Comment:
if any option is more than 20 characters, we show a single column set of checkboxes, otherwise, we lay out small options in 2 columns.

![Screen Shot 2022-08-01 at 6 38 44 PM](https://user-images.githubusercontent.com/30528226/182267602-95c69617-b6f2-497e-ac0e-cd4565f51245.png)
![Screen Shot 2022-08-01 at 6 38 34 PM](https://user-images.githubusercontent.com/30528226/182267605-cf208d3c-6662-48de-8eec-cb1372f8ae65.png)